examples/angular/angular.json

```diff
@@ -25,7 +25,7 @@
             ],
             "styles": [
               "src/styles.css",
-              "node_modules/@stoplight/elements/styles/elements.min.css"
+              "node_modules/@stoplight/elements-dev-portal/styles/elements-dev-portal.min.css"
```

Comment:
That's the only thing that I don't get here. Why is it `elements-dev-portal` here and not `elements`? We still build both from what I can see